### ISSUE TITLE
doesnt launch

### ISSUE THREAD

**Issue by aoehakDoc** [Sigma-Skidder-Team/SigmaRemap#17]
Failed to download file. The file contents differ from what was expected.
Name: lwjgl-openal-3.3.4-natives-macos.jar
URL: https://github.com/MarkGG8181/cloud/raw/refs/heads/main/sigma/lwjgl-openal-3.3.4-natives-macos.jar
Filename on disk: lwjgl-openal-3.3.4-natives-macos.jar
Path: /Users/doot/Library/Application Support/minecraft/libraries/org/lwjgl/lwjgl-openal/3.3.4/lwjgl-openal-3.3.4-natives-macos.jar
Exists: Nonexistent

**Comment by MarkGG8181** [comment]
> Failed to download file. The file contents differ from what was expected. Name: lwjgl-openal-3.3.4-natives-macos.jar URL: https://github.com/MarkGG8181/cloud/raw/refs/heads/main/sigma/lwjgl-openal-3.3.4-natives-macos.jar Filename on disk: lwjgl-openal-3.3.4-natives-macos.jar Path: /Users/doot/Library/Application Support/minecraft/libraries/org/lwjgl/lwjgl-openal/3.3.4/lwjgl-openal-3.3.4-natives-macos.jar Exists: Nonexistent

Please give more specific details, like:
- Launcher;
- If you are using the  official mojang runtime for Java 8;
- Screenshot of the /Users/doot/Library/Application Support/minecraft/libraries/org/lwjgl/lwjgl-openal/3.3.4/ folder;

**Comment by aoehakDoc** [comment]
launcher is official
runtime is official
<img width="772" alt="Screenshot 2024-10-24 at 9 22 10 PM" src="https://github.com/user-attachments/assets/2a255fea-757a-42db-93f9-67584a27cf25">
i downloaded the file manually since it wouldnt do it itself but it still crashes for the same reason

**Comment by MarkGG8181** [comment]
> launcher is official runtime is official <img alt="Screenshot 2024-10-24 at 9 22 10 PM" width="772" src="https://private-user-images.githubusercontent.com/164023998/379786982-2a255fea-757a-42db-93f9-67584a27cf25.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk3NzY1NTIsIm5iZiI6MTcyOTc3NjI1MiwicGF0aCI6Ii8xNjQwMjM5OTgvMzc5Nzg2OTgyLTJhMjU1ZmVhLTc1N2EtNDJkYi05M2Y5LTY3NTg0YTI3Y2YyNS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAyNFQxMzI0MTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jMGE5NzdiMTBlYmFlODVkYzA3MDQ0NTQ1NTRlYzk3NjkzZGU2YTY5MWFmYjg1YTYyNTc4NDg5MTViYjMxZTA3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.PpRHnBTovBFxnTUb0uOctwnTIXY2JpTnBqXTR1FIR5k"> i downloaded the file manually since it wouldnt do it itself but it still crashes for the same reason

Try downloading a custom runtime. Either from oracle or somewhere else. And then you can set it here: 
![image](https://i.imgur.com/J7vqqFN.png)

**Comment by aoehakDoc** [comment]
yea but on mac its not an exe its a bunch of different files
and it makes an error if its path is to a folder instead of a specific file

**Comment by MarkGG8181** [comment]
> yea but on mac its not an exe its a bunch of different files and it makes an error if its path is to a folder instead of a specific file

Yeah, mac’s have a different file type for executables, you set it to javaw.dmg/.app or whatever it’s called.

**Comment by aoehakDoc** [comment]
<img width="1153" alt="Screenshot 2024-10-25 at 6 42 43 AM" src="https://github.com/user-attachments/assets/b30738b6-88ea-4aa6-b16a-514254871e83">

**Comment by aoehakDoc** [comment]
The fifth build prereleased worked without any additions or edits so how come this version doesnt?

**Comment by richylotl** [comment]
> The fifth build prereleased worked without any additions or edits so how come this version doesnt?

Because macos, i'd guess.

**Comment by DataM0del** [comment]
Stale
